<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SonicGLvl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="object_library_tree">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string notr="true"> Object Library</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QScrollArea" name="properties_scroll_area">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>105</width>
              <height>292</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="welcome_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="pixmap">
          <pixmap resource="EditorResource.qrc">:/Editor/Banner</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="version_label">
         <property name="font">
          <font>
           <pointsize>26</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>VERSION</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="open_stage_button">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Open Stage...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Recent</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="recent_list"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="EditorViewportsWidget" name="viewports_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_stage"/>
    <addaction name="separator"/>
    <addaction name="action_save_stage_data"/>
    <addaction name="action_save_stage_terrain"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_pase"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_select_all"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_open_stage">
   <property name="text">
    <string>Open Stage...</string>
   </property>
  </action>
  <action name="action_save_stage_data">
   <property name="text">
    <string>Save Stage Data...</string>
   </property>
  </action>
  <action name="action_save_stage_terrain">
   <property name="text">
    <string>Save Stage Terrain...</string>
   </property>
  </action>
  <action name="actionExport_Scene_as_FBX">
   <property name="text">
    <string>Export Scene as FBX...</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_pase">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionGraphics">
   <property name="text">
    <string>Graphics</string>
   </property>
  </action>
  <action name="actionUse_World_Transform">
   <property name="text">
    <string>Use World Transform</string>
   </property>
  </action>
  <action name="actionUse_Placement_Snap">
   <property name="text">
    <string>Use Placement Snap</string>
   </property>
  </action>
  <action name="actionOpen_Physics_Editor">
   <property name="text">
    <string>Open Physics Editor...</string>
   </property>
  </action>
  <action name="actionDelete_all_Terrain">
   <property name="text">
    <string>Delete all Terrain...</string>
   </property>
  </action>
  <action name="actionGenerate_Terrain_Groups">
   <property name="text">
    <string>Generate Terrain Groups...</string>
   </property>
  </action>
  <action name="actionImport_FBX_as_Terrain">
   <property name="text">
    <string>Import FBX as Terrain...</string>
   </property>
  </action>
  <action name="actionReload_Objects_Template_Database">
   <property name="text">
    <string>Reload Objects Template Database...</string>
   </property>
  </action>
  <action name="actionSave_Objects_Template_Database">
   <property name="text">
    <string>Save Objects Template Database...</string>
   </property>
  </action>
  <action name="actionOpen_Material_Editor">
   <property name="text">
    <string>Open Material Editor...</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditorViewportsWidget</class>
   <extends>QWidget</extends>
   <header>EditorViewportsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EditorResource.qrc"/>
 </resources>
 <connections/>
</ui>
